### PR TITLE
Update replay_perception.py

### PR DESCRIPTION
**Pull request by pinakigupta**
Retaining all the existing functionality. Additionally you can also input multiple objects in the json file now as a list and can be parsed by the replay perception file.
[{
object 1
},
{
object 2
},
{
object 3
}]

### CODE REVIEW

File: modules/tools/perception/replay_perception.py [hunk 1]
```diff
@@ -91,14 +91,23 @@ def load_descrptions(files):
...
-            obstacle = simplejson.loads(fp.read())
-            trace = obstacle.get('trace', [])
-            for i in range(1, len(trace)):
-                if same_point(trace[i], trace[i - 1]):
-                    print('same trace point found in obstacle: %s' %
-                          obstacle["id"])
-                    return None
-            objects.append(obstacle)
+            obstacles = simplejson.loads(fp.read())
+	    if type(obstacles) is list: # Multiple obstacle in one file saves as a list[obstacles]
```
Comment (xiaoxq) [inline modules/tools/perception/replay_perception.py]: Fix indents. No tabs.

Comment (pinakigupta) [reply modules/tools/perception/replay_perception.py]: Fixed indent issues. Please verify.

**Comment by pinakigupta** [comment]
Can someone please close this review ?

**Comment by mickeyouyou** [comment]
It has hanged the logic. each json (that contains id, postion, type)  is a object, not in filed trace.

**Comment by pinakigupta** [comment]
@mickeyouyou  I am not sure if I follow you completely about whats missing in my code. Have you tested the code to see if there are any issues. Of course, as you indicated the obstacles class include the trace, and other fields like"id" etc. I think they are all maintained. 

Here is an example of a json file I created for multiple objects. I didn't think it was necessary to upload it, but maybe I should have uploaded it to make my intentions clear. 

# combined_objects.json
[{
    "id": 2,
    "position": [369193.03, 4321565.05, 0.0],
    "theta": 1.53,
    "length": 4,
    "width": 1.3,
    "height": 1,
    "speed": 0,
    "tracking_time": 10,
    "type": "VEHICLE",
    "trace": [[369193.03, 4321565.05, 0.0]]
},

{
    "id": 4,
    "position": [369063.67, 4321341.47, 0.0],
    "theta": -0.05,
    "length": 4,
    "width": 2,
    "height": 1,
    "speed": 2,
    "tracking_time": 10,
    "type": "VEHICLE",
    "trace": [[369063.67, 4321341.47, 0.0],
    [369067.53, 4321341.27, 0.0],
    [369071.33, 4321341.07, 0.0],
    [369075.02, 4321340.96, 0.0],
    [369078.74, 4321340.93, 0.0]]
}]

**Comment by pinakigupta** [comment]
@xiaoxq could you please see if your concerns are addressed ?

**Comment by pinakigupta** [comment]
@xiaoxq  can you please approve this simple change ?

**Pull request merged**